insonmnia/dwh/sql.go

```diff
@@ -1134,6 +1134,61 @@ func (m *sqlStorage) GetLastEvent(conn queryConn) (*blockchain.Event, error) {
 	}, nil
 }
 
+func (m *sqlStorage) getStats(conn queryConn) (*sonm.DWHStatsReply, error) {
+	var (
+		numCurrDealsQ, _, _ = m.builder().Select("count(Id)").From("Deals").Where("Status=1").Prefix("(").Suffix(")").ToSql()
```

Comment:
maybe pass const with status via `.Where`'s  args?

Reply:
not sure how this is any better, do you have any ideas?

Reply:
ok, sounds reasonable